src/components/Categories.jsx

```diff
@@ -29,4 +22,8 @@ const Categories = () => {
   )
 }
 
+Categories.propTypes = {
```

Comment:
Good job for this!, when not working with TypeScript this is a best practice to keep in mind!